typo "module" (package.json).

package.json
`
"module": "dist/chartjs-plugin-gradient.ems.js"
`
ems -> esm.

so not found module by vite.

`
✘ [ERROR] [plugin vite:dep-scan] Failed to resolve entry for package "chartjs-plugin-gradient". The package may have incorrect main/module/exports specified in its package.json.
`

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "chartjs-plugin-gradient",
-  "version": "0.5.0",
+  "version": "0.5.1",
   "description": "Easy gradient colors for Chart.js",
   "main": "dist/chartjs-plugin-gradient.js",
   "module": "dist/chartjs-plugin-gradient.esm.js",

